homeassistant/components/emulated_hue/hue_api.py

```diff
@@ -393,141 +459,66 @@ def __init__(self, config):
                 hass.services.async_call(domain, service, data, blocking=True)
```

Comment:
Why is this a create_task with blocking=True. In that case you could also just do `async_call` with `blocking=False`.

Reply:
> Why is this a create_task with blocking=True. In that case you could also just do `async_call` with `blocking=False`.

Not sure ( I didn't write that :) ). I will test with blocking=False and report in later with the result.